### PR TITLE
Autoscaling Off peak support

### PR DESCRIPTION
**Pull request by azman0101**

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -89,6 +89,7 @@ gitlab_runner__home: '{{ (ansible_local.root.home
 	                  else "/var/local") + "/" + gitlab_runner__user }}'
 
                                                                    # ]]]
+
```
Comment (drybjed) [inline defaults/main.yml]: This line seems to be unneccesary, can you remove it?

**Review by drybjed — COMMENTED** [review]
Looks good to me. Since the PR is tagged with `[WIP]`, do you plan to add something else, or is this ready for merging?

**Comment by azman0101** [comment]
Hi, I haven't tested before submitting. That's why I've tagged the PR [WIP]

**Comment by drybjed** [comment]
@azman0101 I see, `cache_valid_time` works only with the `apt` module. OK, remove the last commit here, I'll merge the other PR.

**Comment by drybjed** [comment]
@azman0101 Since the PR is marked as `[WIP]`, any progress in testing? if the PR is ready, rename it to indicate that. :-)

**Comment by azman0101** [comment]
Hi @drybjed,

Sorry for the delay. I've tested it

**Review by drybjed — APPROVED** [review]

**Pull request merged**